UITests/Receipts/PrepaidCardListReceiptTests.swift

```diff
@@ -133,13 +145,6 @@ class PrepaidCardListReceiptTests: BaseTests {
         XCTAssertEqual(transactionDetails.receiptIdValue.label, "FISVL_5269000")
         // Will comment out for now
         //XCTAssertEqual(transactionDetails.dateValue.label, "Thu, Jun 20, 2019, 9:23 PM")
-
-        // Fee
-        XCTAssertEqual(transactionDetails.feeSection.label, "Fee Specification")
```

Comment:
Add test case to verify fee section

Reply:
Added verification that fee section doesn't exist.

Reply:
The above comment was to add 1 UI Test to validate Fee Section exists. That UI tests still needs to be added

Reply:
@jsaini-hw Sorry, I still cannot get your point. This test case (as well as the next one) supposes fee section not exists.  
Fee section exists and is being validated within `testPrepaidReceiptDetail_verifyTransactionOptionalFields()`.
Could you please provide me more details on expected test cases?

Reply:
@oleksii-lubianyi-epam as long as it is tested, I am good